---a/net/minecraft/entity/passive/IAnimals.java
+++b/net/minecraft/entity/passive/IAnimals.java
@@ -1,5 +1,5 @@
 package net.minecraft.entity.passive;
 
-public interface IAnimals
-{
-}
+public interface IAnimals {
+
+}
